activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Emit active support events from analyzers.
```

Comment:
I think the changelog message is a bit clearer if we mention `instrumentation`:
```suggestion
*   Emit active support instrumentation events from analyzers.
```

Reply:
Thanks for the suggestion! Committed :)